### PR TITLE
a11y: Accessibility issue

### PR DESCRIPTION
**Pull request by liweitian**
## Task Item
refs #2143 
refs #2123 

## Screenshots
微信图片_20200308230924

### CODE REVIEW

File: Composer/packages/client/src/pages/design/index.tsx [hunk 1]
```diff
@@ -389,6 +389,7 @@ function DesignPage(props) {
                     hidden={triggerButtonVisible || !selected}
                     src={`${rootPath}/extensionContainer.html`}
                     ref={addRef}
+                    title={formatMessage('visual editor')}
```
Comment (corinagum) [inline Composer/packages/client/src/pages/design/index.tsx]: @DesignPolice As a team, do we want to enforce copy rules?
`Visual editor`
also for the changes below: 
`Form editor`,
`Show all notifications`,
`Show ${item} notifications`

Comment (corinagum) [reply Composer/packages/client/src/pages/design/index.tsx]: For reference: https://docs.microsoft.com/en-us/windows/win32/uxguide/text-ui

Comment (corinagum) [reply Composer/packages/client/src/pages/design/index.tsx]: *Looks like it might actually be "Visual Editor". I remember the rules incorrectly :/

Comment (DesignPolice) [reply Composer/packages/client/src/pages/design/index.tsx]: Thanks @corinagum - I think we should be following any and all copy rules. My guess is that we have a mix of going to fast and not one source for copy... which we should have. I will tag @mareekuh on this and maybe we can get Yan to take a pass, unless you are wanting the practice... but, I think copy probably needs some help outside of just the rules.

Comment (corinagum) [reply Composer/packages/client/src/pages/design/index.tsx]: Either way, happy to help out. lmk!

**Pull request merged**